Monitoring in a Box/Ansible/inventory.yaml

```diff
@@ -1,17 +1,5 @@
-### Which Proxmox Node to run the task on ###
-[pve_node]
-pve0
-pve1
-pve2
-pve3
-pve4
-pve5
-
-[pve_node:vars]
-ansible_become_pass=[key in your sudo password]
-
 [mib_host]
-mib@10.0.1.240
+mib1  ansible_user=mib	ansible_host=10.0.1.240
```

Comment:
Can this be part of the hostvar? Have a single source of truth as far as possible.

Reply:
I will shift it to host_var and test if it works

Reply:
I tested it already and it works. I will be using the following set up to link the variables together:
ansible_user: "{{NewVmUsername}}"
ansible_host: "{{NewVmIP}}"

Reply:
Cool 👍

Reply:
I will also be shifting the ssh jumphost argument into the host_vars file to consolidate all the variables which require editing in a single file. Therefore user only needs to access host_vars to fill in all the necessary variables.

Reply:
I have shifted the variables in the `Shift Inventory variables to host_vars file` commit